submission/proxy_server.go

```diff
@@ -45,6 +46,20 @@ func NewProxyServer(logListPath string, dBuilder DistributorBuilder, reqTimeout
 // Run starts regular Log list updates. Blocks until initialization happens.
 func (s *ProxyServer) Run(logListRefreshInterval time.Duration, rootsRefreshInterval time.Duration) {
```

Comment:
Run should accept a context as it's first argument

Reply:
Done